plugins/exilepearl-paper/src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java

```diff
@@ -188,7 +207,39 @@ public void addedToInventory(ClickableInventory inv, int slot) {
 		}
 
 		LazyList<IClickable> lazyContents = MoreCollectionUtils.lazyList(contentSuppliers);
-		new MultiPageView(sender, lazyContents, "All Pearls", true).showScreen();
+		final var pageView = new MultiPageView(sender, lazyContents, "All Pearls", true);
+
+		pageView.setMenuSlot(constructBannedPearlsToggleClick(bannedPearlToggle), 4);
+		pageView.showScreen();
 	}
 
+	private IClickable constructBannedPearlsToggleClick(final boolean bannedPearlToggle) {
+		final var item = new ItemStack(Material.BARRIER);
+		ItemUtils.handleItemMeta(item, (final ItemMeta meta) -> {
+			meta.displayName(Component.text()
+					.decoration(TextDecoration.ITALIC, false)
+					.color(NamedTextColor.GOLD)
+					.content("Toggle banned pearls")
+					.build());
+			MetaUtils.setComponentLore(meta,
+					Component.text()
+							.decoration(TextDecoration.ITALIC, false)
+							.color(NamedTextColor.AQUA)
+							.content("Currently turned " + (!bannedPearlToggle ? "on" : "off"))
+							.build());
+			return true;
+		});
+		return new Clickable(item) {
+			@Override
+			public void clicked(final Player clicker) {
+				if (onCoolDown(clicker)) return;
```

Comment:
Does this mean that the toggle can only clicked 10 seconds after opening the GUI?

Reply:
Yes but it will only matter on the rare occasion someone wants to look at banned pearls